src/components/detailsPage/pages/resources/ResourcesDetailsPage.tsx

```diff
@@ -0,0 +1,74 @@
+import MapCard from 'components/cards/MapCard';
+import SidebarCard from 'components/cards/SidebarCard';
+import { AuthContext } from 'context/AuthContextProvider';
+import { useContext, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { CypherQueryApiService as cypherQuery } from 'services/cypherQueryApiService';
+
+import car from '../../../../assets/car.gif';
+
+import styles from './ResourcesDatails.module.css';
+import ResourcesMainContent from './ResourcesMainContent';
+
+export default function ResourcesDetailsPage() {
```

Comment:
The name should be in singular `ResourceDetailsPage`. You just show the details of a single resource.